polaris-react/tests/utilities/listbox.tsx

```diff
@@ -26,10 +26,7 @@ export function mountWithListboxProvider(
     },
     render(element: React.ReactElement) {
       return (
-        <PolarisTestProvider
-          i18n={translations}
-          features={{newDesignLanguage: true}}
```

Comment:
not sure why this is still here since this was something we removed in v6 (afaik from the docs)